src/nanoemoji/paint.py

```diff
@@ -100,13 +103,13 @@ def breadth_first(self) -> Generator[PaintTraverseContext, None, None]:
             yield context
             transform = context.transform
             # TODO paint variants
-            if isinstance(context.paint, PaintTransform):
-                transform = Affine2D.compose_ltr(
-                    (
-                        transform,
-                        Affine2D(*context.paint.transform),
-                    )
+            paint_transform = context.paint.gettransform()
+            transform = Affine2D.compose_ltr(
+                (
+                    transform,
+                    Affine2D(*paint_transform),
```

Comment:
if the return value from gettransform method is already an Affine2D, why do you need to unpack and construct a new Affine2D?

Reply:
Sheer incompetance :D